msteams-platform/TOC.yml

```diff
@@ -268,6 +268,8 @@
   - name: Get context for your tab
     href: tabs/how-to/access-teams-context.md
     displayName: User context, authentication flow from Microsoft Azure Active Directory, retrieve context, theme changes
+  - name: Navigate within personal tabs
```

Comment:
I think this topic can apply to all tabs (including team/channel scope). Let's verify w/VSR team

Reply:
I will double check this!

Reply:
@erikadoyle @JoshuaPartlow 
Yuri Dogandjiev just informed me that the APIs should work the same between Personal Tabs and Channel Tabs. Let me know if there are other questions

Reply:
[18-11 06:52 am] Joshua Partlow
Hello - I am always happy to connect and talk. Time zones might make it hard to find a good time, but I am happy to try. Or if you want to put your questions in this chat, I can answer when I get in during my morning. I do want to mention that this PR is still a Draft that I'm working on with the engineering team. I created a PR to make the review with them easier, but it's likely it will still be a bit before it's ready. We have a few questions they're investigating that will influence whether it needs to change. Should I have waited until the article was more complete to create the Draft PR?

[18-11 10:13 am] Aman Samuel Prasad (Ushta Te Consultancy Services)
Sure Joshua, I will ping you my queries and please let me know once the draft is ready?
Thanks.

@surbhigupta12 @v-abirade 
Waiting for the doc draft to be completed.

Reply:
Updated to reflect in my upcoming commit.